config/locales/en.yml

```diff
@@ -131,6 +131,14 @@ en:
               inclusion: 'Select whether you have requirements for school experience candidates'
             requirements_details:
               blank: 'Details required'
+        schools/on_boarding/fees:
+          attributes:
+            administration_fees:
+              inclusion: 'Select an opiton'
```

Comment:
`opiton` is spelt as `option` :)

Reply:
On further consideration - can the message be clearer - 'Select Yes or No' ? Thanks

Reply:
I've updated the validation message to include the field so it makes sense in the summary panel